### PR TITLE
chore: fix broken links

### PR DESCRIPTION
**Pull request by yin1999**
### Description

fix broken links in HTML. This is done by the `yarn tool fix-flaws` command (with the [HTML-in-markdown](https://github.com/yin1999/yari/commit/f3bdc2301070f928bc1350323d2d35e24f7486db) support for this tool).

### Related issues and pull requests

- [yari can now detect the broken link flaws in the HTML content from markdown files](https://github.com/mdn/yari/pull/8133)

### CODE REVIEW

File: files/en-us/web/events/index.md [hunk 1]
```diff
@@ -202,10 +202,6 @@ This topic provides an index to the main _sorts_ of events you might be interest
         </p>
         <p>
           Wheel events fired on
-          <a href="/en-US/docs/Web/API/Document/wheel_event"
-            ><code>Document</code></a
-          >
-          and
```
Comment (yin1999) [inline files/en-us/web/events/index.md:208]: As [the target of the `wheel` event](https://w3c.github.io/uievents/#event-type-wheel) is `Element`, and the link has been redirected to the event on Element, I removed this part.

**Review by rebloor — COMMENTED** [review]
OK with regard to the changes on the add-ons/webextensions path

File: files/en-us/web/api/window/sidebar/index.md [hunk 2]
```diff
@@ -69,7 +69,7 @@ The sidebar object returned has the following methods:
         <p>
           Installs a search engine (Sherlock).
           <a
-            href="/en-US/docs/Web/API/Window/sidebar/Adding_search_engines_from_Web_pages#Installing_Sherlock_plugins"
+            href="/en-US/docs/Web/OpenSearch#installing_sherlock_plugins"
             title="Adding_search_engines_from_web_pages"
```
Comment (estelle) [inline files/en-us/web/api/window/sidebar/index.md:73]:
```suggestion
```

**Review by estelle — COMMENTED** [review]
The /web/ section is approved.
I made a suggestion to remove a title from a link.

**Review by estelle — APPROVED** [review]
Both directories are reviewed and approved. Thanks!

**Pull request merged**